docs/kola/external-tests.md

```diff
@@ -103,6 +103,52 @@ it out.
 (Previously the API for this was to send `SIGTERM` to the current process; that
 method is deprecated and will be removed at some point)
 
+## HTTP Server
+
+The `kolet` binary is copied into the CoreOS system running the tests. Notably,
+it contains the built-in command `kolet httpd [port] [path]` for starting an
+HTTP file server at port to serve the contents of the file system rooted at path.
+
+To have an HTTP server started before your tests runs, you can create an Ignition
+config (or Fedora CoreOS config) and include it in your external test directory.
+
+In the following Fedora CoreOS config example, the Ignition config includes a
+path unit and a service unit. The path unit ensures that the httpd service runs
+automatically once the Kolet binary is copied to the system. An HTTP server will
+be started at `localhost:8080` and serve the files in `/var/home/core`. Your test
+can then do e.g. `curl localhost:8080/hello_world.txt`.
+
+Example:
+```
+variant: fcos
+version: 1.1.0
+systemd:
+  units:
+    - name: kolet-httpd.path
+      enabled: true
+      contents: |
+        [Path]
+        PathExists=/var/home/core/kolet
+        [Install]
+        WantedBy=multi-user.target
+    - name: kolet-httpd.service
+      contents: |
+        [Service]
+        ExecStart=/var/home/core/kolet httpd 8080 /var/home/core -v
+        [Install]
+        WantedBy=multi-user.target
+storage:
+  files:
+    - path: /var/home/core/hello_world.txt
+      contents:
+        inline: Hello, world!
+      mode: 0644
+      user:
+        id: 500
```

Comment:
Where'd you get `500`?  Hmm, it'd be nice if Ignition had a feature where it automatically used the user's uid/gid when writing to a homedir, but today it looks like you can use `name: core` instead.

That said the test code runs as root, so it could just be `/var/tmp/my-kola-test-data` or so.

Reply:
I just copied it from one of the [examples](https://coreos.github.io/fcct/examples/#files) from the fcct docs :)